screen_tabs.py

```diff
@@ -85,11 +86,26 @@ class ScreenTabsPrefs(AddonPreferences):
 		min = 0.5,
 		max = 1.0)
 
+	screen_text_length = IntProperty(
+		name = 'Screen Tab Text Length',
+		description = "Maximum number of characters to display per screen tab",
+		default = -1,
+		min = -1,)
+
+	screen_text_scale = FloatProperty(
```

Comment:
If length is very small then user will see:
![default](https://user-images.githubusercontent.com/32243930/45771748-fd703a00-bc56-11e8-8efa-001315a88986.png)

Reply:
Also, the default length value should be 1 and max value 1 as well. No reason for double sized tabs.

Reply:
Icons + small scale, plus no text could look like this:  
![z1](https://user-images.githubusercontent.com/10817631/45773056-f45b7a80-bc0e-11e8-9a1a-7ea9678220a5.png)

Same scale + all text, could then look like this: 
![z2](https://user-images.githubusercontent.com/10817631/45773078-00dfd300-bc0f-11e8-8217-ed4d951f1edb.png)

Same scale + 10 characters: 
![z3](https://user-images.githubusercontent.com/10817631/45773203-4d2b1300-bc0f-11e8-91d2-0e7fa54768e8.png)

Scale is set to 0.40

Scale 0 = disabled. 
Negative values hide the buttons and pushes the rest of the row left (it's weird).

Reply:
> Also, the default length value should be 1 and max value 1 as well. No reason for double sized tabs.

I have no objection to a soft_max of 1 and default, but the reason I don't specify a max is to allow people to increase however they want.   like this: 
![z4](https://user-images.githubusercontent.com/10817631/45773717-9e87d200-bc10-11e8-8868-0ca6ba3f1f5d.png)

Not something "I" would do but it leave the choice. 
I didn't think about 1 as a default but I'll change stuff around to make it a percentage.

Reply:
Using 0 value as disabled is way too bad because it is length variable so 0 means 0 length. We don't need to disable or check anything, just use length 1 as default.